modules/jfx.incubator.richtext/src/main/java/jfx/incubator/scene/control/richtext/LineEnding.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package jfx.incubator.scene.control.richtext;
+
+/**
+ * Specifies line separator (line ending) characters.
+ *
+ * @since 26
+ */
+public enum LineEnding {
+    /** Classic Mac OS line ending, ASCII CR (0x0d) */
+    CR,
+    /** Windows line ending, ASCII LF (0x0a) */
+    CRLF,
+    /** macOS/Unix line ending, sequence of CR/LF (0x0d 0x0a)*/
+    LF,
+    /** Platform default line ending, using the value of {@code System.getProperty("line.separator")} */
+    SYSTEM_DEFAULT;
```

Comment:
or maybe just `SYSTEM`? INow that I look at it, 'm not sure we need the "DEFAULT" suffix (nor the word "default" in the description).